contracts/Managers/RoleManager.sol

```diff
@@ -11,6 +11,16 @@ import {GenericDebtAllocatorFactory} from "../debtAllocators/GenericDebtAllocato
 
 /// @title Yearn V3 Vault Role Manager.
 contract RoleManager is Governance2Step {
+    /// @notice Emitted when a new vault has been deployed or added.
+    event AddedNewVault(
+        address indexed vault,
+        address indexed debtAllocator,
+        uint256 rating
```

Comment:
It would be used once this is deployed, both for [naming conventions](https://github.com/yearn/vault-periphery/blob/3cbfec6e618be9482969687e72985f6059d87549/contracts/Managers/RoleManager.sol#L245-L253) as well as on chain tracking.

Also for future use by a strategy manager contract to potentially compare rating before adding certain strategies.